### ISSUE TITLE
[1.0.6.1] Bugs that need urgent fixation

### ISSUE THREAD

**Issue by ansinerd** [emuPlace/emuThreeDS#95]
Device : iPhone 13 & iPad pro 12.9 inch

issues : the opaque thumb stick is extremely bothering, it hides so much screen space on iPhone display that sometime you just cannot play games. Obstructive nature. 

Despite everyone’s dislike for EGG3DS I still wish and admire their on screen transparent control which is slick, not obstructive and functional with all buttons.

2. You cannot play games like Metroid Ramus or even Shinobi returns because of lack of extra buttons like R and L. 

3. The notch issue isn’t resolved as well for mobile device.

iPad pro 12.9 inch specific bugs : 
4. The app isn’t optimised for bigger display, its smaller phone version upscaled for larger device, it feels wrong when using.
5. Much screens space is wasted (black bars)
6. The opaque thumbstick feels even more obstructive 
7. RAM LEAK ISSUE : even M1 chip iPad pro with 8gb of RAM, cannot seem to keep the emulator in background, i tested 10 times that even if by mistake you minimise the app, the emulator resets itself.

common bugs : 
No option to close the rom without closing the emulator and relaunching with JIT.
6. Lack of HLE mode in settings makes it kinda useless for those who are travelling and do not have access to wifi to enable JIT. 
7. The setting menu is not accessible 
8. The click does not register in any part in the app

In conclusion, the app feels good, but many deal breakers as of current version which needs fixing soon.

Screen layout optimisation should be priority for bigger devices.
RAM leakage issue shall be addressed if possible.
Proper touch controls lay out need to be implemented. I personally feel there should be no shame in even copying EGG3DS touch layout, that feels much better or if not, at least allow skin for touch controllers like Delta, so that someone can come up with better on screen touch controls.

rest the emulator port itself is going in right direction and with constant refinement to original citra, we can only expect better performance in coming days.
